go.mod

```diff
@@ -1,19 +1,22 @@
 module github.com/rook/kubectl-rook-ceph
 
-go 1.21
+go 1.22.0
```

Comment:
Let's make the toolchain update with a separate PR. @subhamkrai Want to take care of that?